vendor/cpp-netlib/conanfile.py

```diff
@@ -14,51 +20,54 @@ class CppNetlib(ConanFile):
     options = {
         "shared": [True, False],
     }
+    no_copy_source = True
     default_options = {
         "shared": False,
     }
-    exports_sources = [
-        "CMakeLists.txt",
-    ]
-    generators = "cmake"
     requires = [
         # CppNetlib does not work with a boost that is newer than 1.69
         "boost/[>=1.65.0 <1.70]",
-        "openssl/1.1.1g",
+        "openssl/1.1.1s",
```

Comment:
Do we have to re-upload this package internally? As far as I can see we have only sources uploaded and no binaries.

Reply:
This is likely a recommended security update. If your company is hosting Conan packages internally, I recommend uploading binary packages whenever you make a release.